synapse/cortex.py

```diff
@@ -2881,7 +2875,7 @@ async def _normStormPkg(self, pkgdef, validstorm=True):
         for configvar in pkgdef.get('configvars', ()):
             self._reqStormPkgVarType(pkgname, configvar.get('type'))
 
-    async def loadStormPkg(self, pkgdef):
+    def loadStormPkg(self, pkgdef):
```

Comment:
Maybe a comment on this and `_dropStormPkg()` about it not being async on purpose and if it needs to be made async then someone needs to fix the race condition another way.